### PR TITLE
Issue #1228 feat: Math text plugin renderer

### PR DESCRIPTION
**Pull request by AkshayaRathinavel**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "editor":
+  {
+    "main": "editor/plugin.js",
+    "dependencies": [
+    {
+      "type": "js",
+      "src": "editor/init.js"
+    }],
+    "menu": [],
+    "help":
```
Comment (manojchandrashekar) [inline manifest.json]: Remove `help` parameter.

File: renderer/plugin.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// Renderer plugin can't be tested as of now
+// Please move the logic to other classes and test them independently
+// Let the plugin class delegate functionality to these classes
+Plugin.mathtext = {};
+
+Plugin.mathtext.RendererPlugin = Plugin.extend({
```
Comment (manojchandrashekar) [inline renderer/plugin.js]: Don't use `Plugin.mathtext`. Use similar to:

```
org.ekstep.questionunitmtf = {};
org.ekstep.questionunitmtf.RendererPlugin = org.ekstep.contentrenderer.questionUnitPlugin.extend({
```

File: editor/init.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Initialize the plugin namespace
```
Comment (manojchandrashekar) [inline editor/init.js]: Don't use init.js. Create namespace in plugin.js itself.

File: manifest.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  "ver": "1",
+  "author": "ekstep",
+  "type": "plugin",
+  "title": "mathtext",
+  "description": "Math text standalone plugin",
+  "editor":
+  {
+    "main": "editor/plugin.js",
+    "dependencies": [
+    {
```
Comment (manojchandrashekar) [inline manifest.json]: Remove init.js and it's dependency here.

File: renderer/plugin.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  _render: true,
+  initPlugin: function(data) {
+    var pluginData;
+
+    if (!_.isUndefined(data.data))
+      pluginData = JSON.parse(data.data.__cdata);
+
+    var pid = data._id || data.id;
+
+    this.id = _.uniqueId('org.ekstep.mathtext');
```
Comment (manojchandrashekar) [inline renderer/plugin.js]: Why `_.uniqueId` is used here?

File: renderer/plugin.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    var dims = this.relativeDims();
+    var div = document.getElementById(data.id);
+    if (div) {
+      jQuery("#" + data.id).remove();
+    }
+    div = document.createElement('div');
+    if (mathData.style)
+      div.setAttribute("style", mathData.style);
+    div.id = mathData.id;
+    div.classList.add('math-text');
```
Comment (manojchandrashekar) [inline renderer/plugin.js]: Use constants

Comment (AkshayaRathinavel) [reply renderer/plugin.js]: Constants cannot be used, because div element has to be created dynamically for each equation.

**Comment by AkshayaRathinavel** [comment]
Changes are merged with new mathtext editor

**Pull request closed without merging**